locales-he-IL.xml

```diff
@@ -23,29 +23,29 @@
     <term name="and">ו</term>
     <term name="and others">ואחרים</term>
     <term name="anonymous">אלמוני</term>
-    <term name="anonymous" form="short">anon</term>
+    <term name="anonymous" form="short"></term>
     <term name="at">-ב</term>
     <term name="available at">זמין ב</term>
     <term name="by">על-ידי</term>
     <term name="circa">לערך</term>
-    <term name="circa" form="short">c.</term>
+    <term name="circa" form="short">~</term>
     <term name="cited">מצוטט ב</term>
     <term name="edition">
       <single>מהדורה</single>
       <multiple>מהדורות</multiple>
     </term>
-    <term name="edition" form="short">ed</term>
+    <term name="edition" form="short">מהדורה</term>
     <term name="et-al">ואחרים</term>
     <term name="forthcoming">צפוי</term>
```

Comment:
In Hebrew journals, I have not seen a differentiation between "forthcoming" and "in press", both using בדפוס. Those who do differentiate would use בקרוב not צפוי